src/store/store.ts

```
@@ -402,22 +418,111 @@ export const useGameStore = create<GameState>((set, get) => ({
   isLivingDomeWindowVisible: false, // Initially hidden
   // --- NEW: Initial Production Dome State ---
   isProductionDomeWindowVisible: false, // Initially hidden
+  // --- NEW: Initial Tutorial State ---
+  isTutorialWindowVisible: false, // Initially hidden
```

Biome (1.9.4)

[error] This property value named isTutorialWindowVisible is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named isTutorialWindowVisible


(lint/suspicious/noDuplicateObjectKeys)